recreate_test.go

```diff
@@ -0,0 +1,206 @@
+//+build integration
+//+build scylla
+
+// Copyright (C) 2017 ScyllaDB
+
+package gocql
+
+import (
+	"io/ioutil"
+	"sort"
+	"strings"
+	"testing"
+	)
+func TestRecreateSchema(t *testing.T) {
+	session := createSessionFromCluster(createCluster(), t)
+	defer session.Close()
+
+	tcs := []struct {
+		Name     string
+		Keyspace string
+		Input    string
+		Golden   string
```

Comment:
Input and Golden should be and are the same!

Reply:
Not really, scylla assigns default values to table options, user usually doesn't specify them, but they should always be in recreate output.